nomos-da/full-replication/src/attestation.rs

```diff
@@ -0,0 +1,70 @@
+use bytes::Bytes;
+use nomos_core::{
+    da::{
+        attestation::{self, Attestation as _},
+        auth::Signer,
+        blob,
+    },
+    wire,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::{hash, Blob, Voter};
+
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+#[cfg_attr(feature = "openapi", derive(utoipa::ToSchema))]
+pub struct Attestation {
+    blob: [u8; 32],
+    voter: Voter,
+    sig: Option<Vec<u8>>,
```

Comment:
Why is this signature optional?

Reply:
During development this implementation had attestation signed separately. Agree that it might not be necessary to have it optional - I still need to add serialization, fill fix it there if unnecessary.